.github/workflows/ubuntu_clean_meson_build.yml

```diff
@@ -0,0 +1,27 @@
+name: Minimal meson build in Ubuntu
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-python@v1
+    - name: install minimal requirements
+      run: sudo apt-get update && sudo apt-get install -y libglib2.0-dev libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev
+    - uses: BSFishy/meson-build@v1.0.3
+      with:
+        action: build
+      with:
+        action: install
```

Comment:
Is ```install``` required?

Reply:
No. I just wanted to test if "ninja -C build install" has no problem in macos.